src/main/io/serial_cli.c

```diff
@@ -1939,6 +1962,82 @@ static void cliFeature(char *cmdline)
     }
 }
 
+#ifdef BEEPER
+static void cliBeeper(char *cmdline)
+{
+    uint32_t i;
```

Comment:
`i` is sometimes redeclared in inner scope, sometimes this declaration is used. It should be consistent (preferably deleting this declaration)